### PR TITLE
Html file customization

### PR DESCRIPTION
**Pull request by RoseGabriely**
## Description

Our website's tab was styled with the default title and icon that came with the HTML file. I changed the title to "CoderHeros" and updated the logo to the CoderHeros logo from our figma document.

## Loom Video

[Loom Video](https://www.loom.com/share/a295afdd4e5d408e829ad36d961d988c)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by ElchinNasirov — APPROVED** [review]
Quick but important changes. Well done.

**Review by seanmp3 — CHANGES_REQUESTED** [review]
Good meta data update, its nice seeing how much simple things adds to the look of the website

File: public/index.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-  <title>Basic SPA</title>
+  <title>CoderHeros</title>
```
[the remark this answers is not in the document]
Comment (RoseGabriely) [reply public/index.html]: Great catch! Fixed!

**Review by seanmp3 — APPROVED** [review]
Perfectly done, thanks Rose for all your work as a tpm this unit

**Comment by manny-glez** [comment]
Well done, its looks better already.

**Review by manny-glez — APPROVED** [review]
Well done, its looks better already.

**Pull request merged**